src/style.css

```diff
@@ -1 +1,15 @@
-/* styles go here */
+.labeles {
+  display: block;
+  margin-top: 10px;
+}
+
+.blocks {
```

Comment:
The class name 'blocks' is too generic. Consider using a more descriptive name, like 'form-blocks', for better understanding of its purpose 🎯